test/IRGen/conditional_conformance_execute.swift

```diff
@@ -0,0 +1,33 @@
+// RUN: %empty-directory(%t)
+// The file that's `main` needs to be called that.
+// RUN: cp %s %t/main.swift
```

Comment:
REQUIRES: executable_test, should go into test/Interpreter/.

You can put common files for conditional conformance tests in test/Inputs/